app/helpers/skins_helper.rb

```diff
@@ -51,4 +50,24 @@ def show_advanced_skin?(skin)
         !skin.skin_parents.empty?)
   end
 
+  def my_site_skins_link(link_text = nil)
+    link_text ||= "My Site Skins"
+    span_if_current ts(link_text), user_skins_path(current_user, skin_type: "Site"), current_page?(user_skins_path(current_user)) && (params[:skin_type].blank? || params[:skin_type] == "Site")
```

Comment:
It was a pain in the ass having to include this whole long thing everywhere we used the My Site Skins link (ditto the Public Site Skins link), so I moved it to the helper. I dragged the work skins links along for the hell of it, even though they're shorter.